src/index.ts

```diff
@@ -55,6 +55,10 @@ export type { DatabaseModule } from './modules/database';
 export type { DatatypeModule } from './modules/datatype';
 export type { DateModule, SimpleDateModule } from './modules/date';
 export type { Currency, FinanceModule } from './modules/finance';
+export type {
+  BitcoinAddressFamilyType,
+  BitcoinNetworkType,
+} from './modules/finance/bitcoin';
```

Comment:
In addtion to the types we usually make the enums publically available as well. Or did I miss a comment which states that we don't want to do this?

```suggestion
export type {
  BitcoinAddressFamilyType,
  BitcoinNetworkType,
} from './modules/finance/bitcoin';
export {
  BitcoinAddressFamily,
  BitcoinNetwork,
} from './modules/finance/bitcoin';
```